panacea/grpc_client.go

```diff
@@ -19,6 +21,8 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+const pageLimit = 30
```

Comment:
Has there any reason for setting `pageLimit` 30?

Reply:
There is no specific reason to set 30. It's a temp value :)